### ISSUE TITLE
API: direct database access

### ISSUE THREAD

**Issue by chouseknecht** [ansible/galaxy_ng#7]
Post migration to `galaxy_ng`, replace API calls to Pulp with database queries. Since Galaxy and Pulp exist in the same database using SQL will be more efficient and more flexible.

**Comment by chouseknecht** [comment]
Blocked by #9

**Comment by chouseknecht** [comment]
Duplicate of #472
